package.json

```diff
@@ -42,7 +42,7 @@
   "dependencies": {
     "debug": "^2.1.1",
     "express": "^4.12.3",
-    "feathers-commons": "^0.2.0",
+    "feathers-commons": "ebourmalo/feathers-commons#feat-advanced-routes",
```

Comment:
Should be replaced with `feathers-commons": "^0.3.0"` if the corresponding PR is validated and merged (https://github.com/feathersjs/feathers-commons/pull/2)